src/tests/ftest/util/osa_utils.py

```diff
@@ -163,47 +178,69 @@ def verify_single_object(self):
         self.obj.close()
         self.container.close()
 
-    def ior_thread(self, pool, oclass, api, test, flags, results):
+    def run_ior_thread(self, action, oclass, test):
+        """Start the IOR thread for either writing or
+        reading data to/from a container.
+        Args:
+            action (str): Start the IOR thread with Read or
+                          Write
+            oclass (str): IOR object class
+            test (list): IOR test sequence
+            flags (str): IOR flags
+        """
+        if action == "Write":
+            flags = self.ior_w_flags
+        else:
+            flags = self.ior_r_flags
+
+        # Add a thread for these IOR arguments
+        process = threading.Thread(target=self.ior_thread,
+                                   kwargs={"pool": self.pool,
+                                           "oclass": oclass,
+                                           "test": test,
+                                           "flags": flags)
```

Comment:
(pylint-syntax-error) closing parenthesis ')' does not match opening parenthesis '{' on line 203 (<unknown>, line 206)